lib/logging/CLogger.h

```diff
@@ -134,13 +134,13 @@ class DLL_LINKAGE CLogFormatter
 {
 public:
 	CLogFormatter();
-	CLogFormatter(const CLogFormatter & copy);
-	CLogFormatter(CLogFormatter && move);
+	CLogFormatter(const CLogFormatter & copy) = default;
+	CLogFormatter(CLogFormatter && move) = default;
 
-	CLogFormatter(const std::string & pattern);
+	CLogFormatter(std::string pattern);
 
-	CLogFormatter & operator=(const CLogFormatter & copy);
-	CLogFormatter & operator=(CLogFormatter && move);
+	CLogFormatter & operator=(const CLogFormatter & copy) = default;
+	CLogFormatter & operator=(CLogFormatter && move) = default;
```

Comment:
Can be removed I think?
```suggestion
```

Reply:
removed both constructors and operators.